### ISSUE TITLE
Focus indicator is not visible on 'vimeo' link.

### ISSUE THREAD

**Issue by priysn** [Microsoft/BotFramework-WebChat#1373]
**Actual Result:**
Focus indicator is not visible on 'vimeo' link.

**Expected Result:**
Focus indicator should be visible on 'vimeo' link.

**Repro Steps:**
1. Open URL [https://microsoft.github.io/BotFramework-WebChat/full-bundle/](https://microsoft.github.io/BotFramework-WebChat/full-bundle/) in Edge browser.
2. Navigate to 'Show a video attachment from Vimeo' button and activate it.
3. Video will appear at the bottom of the page.
4. Navigate the video using tab key and try to reach at 'vimeo' link.
5. Observe if focus indicator is visible or not on 'vimeo' link.

**User Impact:**
If focus indicator is not visible on any control then it will be very difficult for keyboard dependent user to navigate and access that control.

**Test Environment:**
OS: Windows 10
OS Build: 17763.107
OS Version: 1809
Browser: Edge

**MAS Reference:**
[MAS 2.4.7 - Focus Visible (05A)](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={3b98b04f-4fab-4d3e-81af-a0906a34849c})

**Attachment**
[MAS2.4.7_FocusVisible.zip](https://github.com/Microsoft/BotFramework-WebChat/files/2594515/MAS2.4.7_FocusVisible.zip)

**Comment by compulim** [comment]
This is not a bug from Web Chat. The video content is an IFRAME owned by Vimeo.

**Comment by compulim** [comment]
Reopening so we can confirm from the accessibility team.

**Comment by corinagum** [comment]
Confirmed with accessibility team as external issue

**Comment by Amit8527510735** [comment]
#HCL;#Accessibility;#HCL_BotFramework_WebChat;A11yMAS;WCAG2.4.7
